### PR TITLE
Spell checked and more descriptions added 

### PR DESCRIPTION
**Pull request by OrchardPerformance**
Spell checked and added some more descriptions

### CODE REVIEW

**Comment by rusefi** [comment]
Can one of the admins verify this patch?

**Comment by rusefi** [comment]
@OrchardPerformance git says there are conflicts and cannot be merged? :(

File: firmware/integration/rusefi_config.txt [hunk 1]
```diff
@@ -15,18 +15,22 @@
...
-!    rename the bit or subsctitute unused intereger with any new fields of the same size
+!    rename the bit or substitute unused integer with any new fields of the same size
 !    invoke gen_config.bat to apply the tools which would generate .h and .ini files
 !
 !
 ! each field is declared as
 ! type name;comment
 
 
+<<<<<<< HEAD
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt]: here's the merge conflict

File: firmware/integration/rusefi_config.txt [hunk 2]
```diff
@@ -721,9 +738,14 @@ bit vvtDisplayInverted
...
 	bit tachPulseDurationAsDutyCycle
-	bit isAlternatorControlEnabled;+This enables smart alternator control and activates the extra alternator settings
+	bit isAlternatorControlEnabled;+This enables smart alternator control and activates the extra alternator settings.
+<<<<<<< HEAD
+	bit invertPrimaryTriggerSignal;+This setting flips the signal from the primary engine speed sensor.
+	bit invertSecondaryTriggerSignal;+This setting flips the signal from the secondary engine speed sensor.
+=======
 	bit invertPrimaryTriggerSignal
 	bit invertSecondaryTriggerSignal
+>>>>>>> ff39a7263338c337357e825c2b235b60c54dea0c
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt]: here's another

File: firmware/integration/rusefi_config.txt [hunk 3]
```diff
@@ -747,25 +769,30 @@ uint8_t[3] unusedh;
...
 
-	bit isInjectionEnabled;+Enable fuel injection - This is default off on new projects as a safty feature, set to "true" to enable fuel injection and futher injector settings. 
-	bit isIgnitionEnabled;+Enable ignition - This is default off on new projects as a safty feature, set to "true" to enable ignition and futher ignition settings.
+<<<<<<< HEAD
+	bit isInjectionEnabled;+Enable fuel injection - This is default off for new projects as a safety feature, set to "true" to enable fuel injection and further injector settings. 
+	bit isIgnitionEnabled;+Enable ignition - This is default off for new projects as a safety feature, set to "true" to enable ignition and further ignition settings.
+=======
+	bit isInjectionEnabled;+Enable fuel injection - This is default off for new projects as a safety feature, set to "true" to enable fuel injection and further injector settings. 
+	bit isIgnitionEnabled;+Enable ignition - This is default off on new projects as a safety feature, set to "true" to enable ignition and further ignition settings.
+>>>>>>> ff39a7263338c337357e825c2b235b60c54dea0c
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt]: here's another

File: firmware/integration/rusefi_config.txt [hunk 4]
```diff
@@ -1007,10 +1034,14 @@ float[CRANKING_ADVANCE_CURVE_SIZE] crankingAdvance    ;Optional timing advance t
...
-	int16_t coastingFuelCutClt;+Fuel cutoff is deactivated below this coolant threashold.;"C",        1,     0,  -100,    100,  0
+	int16_t coastingFuelCutClt;+Fuel cutoff is deactivated below this coolant threshold.;"C",        1,     0,  -100,    100,  0
 	
 	int16_t pidExtraForLowRpm;+Increases PID reaction for RPM<target by adding extra percent to PID-error;"%",        1,     0,  0,    100,  0
-	int16_t coastingFuelCutMap;+MAP value above which coasting fuel cut is dissabled to aid piston cooling;"kPa", 1, 0, 0, 250, 0
+<<<<<<< HEAD
+	int16_t coastingFuelCutMap;+MAP value above which coasting fuel cut is disabled to aid piston cooling;"kPa", 1, 0, 0, 250, 0
+=======
+	int16_t coastingFuelCutMap;+MAP value above which coasting fuel cut is disabled to aid piston cooling;"kPa", 1, 0, 0, 250, 0
+>>>>>>> ff39a7263338c337357e825c2b235b60c54dea0c
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt]: last one

Comment (mck1117) [reply firmware/integration/rusefi_config.txt]: Also, this isn't what this setting is for.  This is the MAP at which we reintroduce fuel (ie, you put your foot back on the pedal)

File: firmware/integration/rusefi_config.txt [hunk 5]
```diff
@@ -774,9 +801,9 @@ bit unused_1484_bit_21
 
 
 	int16_t idlePidRpmUpperLimit;+Relative to the target idle RPM;"RPM",      1,      0,       0, 9000,      0
-    int16_t primeInjFalloffTemperature;Prime injection pulse falloff temperature threshold, in Celsuis;"*C",    1,      0,       0,  1000.0,        0
+    int16_t primeInjFalloffTemperature;+This sets the temperature above which no priming pulse is used, The value at -40 is reduced until there is no more priming injection at this temperature.;"*C",    1,      0,       0,  1000.0,        0
 
-	adc_channel_e acSwitchAdc;A/C button input handled as analog input
+	adc_channel_e acSwitchAdc;A/C button input handled as analogue input
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt]: up for debate whether this is a spelling mistake 😛

**Pull request merged**